### PR TITLE
Feature/add extra labels and annotations locust

### PR DESCRIPTION
**Pull request by migueleliasweb**
This PR allows people to customize labels and annotations. This is primarily usefull for people who have kubernetes plugins that rely on custom annotations and/or labels to find resources throught the api.

### CODE REVIEW

**Comment by migueleliasweb** [comment]
/retest

Edit: :cry:

**Comment by migueleliasweb** [comment]
Hey @mattfarina, can you have a quick look at this ? Cheers !

File: stable/locust/values.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ service:
   externalPort: 8089
   internalPort: 8089
   nodePort: 0
+  annotations: {}
+  extra_labels: {}
```
Comment (unguiculus) [inline stable/locust/values.yaml]: Please use camel case.

File: stable/locust/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: locust
 description: A modern load testing framework
-version: 0.1.4
+version: 0.1.5
```
Comment (unguiculus) [inline stable/locust/Chart.yaml]: Please bump the minor version since you added a new feature.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/locust/values.yaml (hunk 1 above)
Comment (migueleliasweb) [reply stable/locust/values.yaml]: :+1:

File: stable/locust/Chart.yaml (hunk 2 above)
Comment (migueleliasweb) [reply stable/locust/Chart.yaml]: :+1:

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/locust/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: locust
 description: A modern load testing framework
-version: 0.1.4
+version: 0.2.4
```
Comment (unguiculus) [inline stable/locust/Chart.yaml]: Should be 0.2.0.

**Pull request merged**